### ISSUE TITLE
FormBuilderTextField keeping focus when trying to select menu on FormBuilderDropdown

### ISSUE THREAD

**Issue by EricPreventa** [danvick/flutter_form_builder#146]
Experiencing a common issue on Flutter textfields and drop down menus illustrated in this issue section: https://github.com/flutter/flutter/issues/22075

Using FormBuilderTextField and FormBuilderDropdown and when the textfield is selected and in-focus, it won't allow me to select the menu of the dropdown. 

I didn't experience this issue before, not sure what changed.

**Comment by mbdax034** [comment]
The current solution for this probleme is to change **buttonTheme** and 
**textTheme** in the MaterialApp
for me 
```dart
theme: ThemeData(
        buttonTheme: ButtonThemeData(minWidth: 10,),
        textTheme: TextTheme(button: TextStyle(fontSize: 10)),
)
```

**Comment by Purus** [comment]
I can confirm on this issue. The issue ocurs even for the Checkbox groups.

This is a blocker for me.

**Comment by danvick** [comment]
Hi @EricPreventas/ @Purus 
May I know which version of the library you're using? Is it the latest?

I'm asking because I thought this issue was already fixed.

**Comment by Purus** [comment]
I was using the latest at that point of time when i posted this message
here.

**Comment by Pannkaaka** [comment]
Same when tapping on a FormBuilderRadio option.

**Comment by danvick** [comment]
This was a Flutter issue but now fixed. Feel free to reopen if not.

**Comment by EricPreventa** [comment]
What Flutter version fixed it? I'm on "Tools • Dart 2.9.0 (build 2.9.0-14.1.beta)" and it seems like the issue has cropped back up with the latest version. 

When I click on a dropdown button it and view the downdown options (all works fine), the issue is when I click on a value. It jumps back to the last textformfield. 

This wasn't an issue until just recently updating to 3.11.6
